apps/codeforafrica/src/lib/api.netlify-cms/utils.js

```diff
@@ -0,0 +1,44 @@
+import fs from "fs";
+import { join } from "path";
+
+import matter from "gray-matter";
+import { marked } from "marked";
+
+import mappings from "./mappings";
+
+export const getCollectionSlugs = (collectionDir) => {
+  return fs.readdirSync(collectionDir);
+};
+
+export const getCollectionBySlug = (collectionDir, slug, fields = []) => {
+  const realSlug = slug.replace(/\.md$/, "");
+  const fullPath = join(collectionDir, `${realSlug}.md`);
+  const fileContents = fs.readFileSync(fullPath, "utf8");
+  const { data, content } = matter(fileContents);
+  const items = {};
+
+  fields.forEach((field) => {
+    if (field === "content") {
+      items.description = marked(content);
+    } else {
+      items[field] = data[field] || "";
```

Comment:
```suggestion
      items[field] = data[field] || null;
```

Or is there a reason to prefer `""`?